subsystem/infra_env_test.go

```diff
@@ -204,4 +204,18 @@ var _ = Describe("Infra_Env", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(script).To(HavePrefix("#!ipxe"))
 	})
+	It("fails to update static network config", func() {
```

Comment:
Minor, but this should say when it fails. For example:
`fails when given invalid static network config`

Reply:
Thank you!